packages/core/src/api/index.ts

```diff
@@ -34,6 +36,8 @@ export class API {
 
 	public readonly invites: InvitesAPI;
 
+	public readonly oauth2: OAuth2API;
```

Comment:
Shouldn't the property name be `oAuth2`?

Reply:
While this is _technically_ correct, the word is most often used as a noun so we'll keep it as-is.